modules/@apostrophecms/task/index.js

```diff
@@ -155,7 +158,7 @@ module.exports = {
             };
             span.setAttribute(telemetry.Attributes.ARGV, telemetry.stringify(argv));
             self.apos.argv = argv;
-            await task.task(argv);
+            result = await task.task(argv);
```

Comment:
So tasks can have a return value on success now, although that is only accessible if they are invoked programmatically rather than at the command line. OK.